content-src/components/TopSites/TopSite.jsx

```diff
@@ -172,12 +173,16 @@ export class TopSiteLink extends React.PureComponent {
       smallFaviconStyle = { backgroundImage: `url(${tippyTopIcon})` };
     } else if (link.customScreenshotURL) {
       // assume high quality custom screenshot and use rich icon styles and class names
+
+      // TopSite spoc experiment only
+      const spocImgURL = link.type === "SPOC" ? link.customScreenshotURL : "";
+
       imageClassName = "top-site-icon rich-icon";
       imageStyle = {
         backgroundColor: link.backgroundColor,
         backgroundImage: hasScreenshotImage
           ? `url(${this.state.screenshotImage.url})`
-          : "none",
+          : `url(${spocImgURL})`,
```

Comment:
@Mardak I believe concern [raised here](https://github.com/mozilla/activity-stream/pull/5265#issuecomment-523109143), doesn't apply in above case when we specifically use customScreenshotURL to display a non-pinned spoc tile. Is that correct?